qa/L0_optional_input/optional_input_test.py

```diff
@@ -270,10 +273,38 @@ def test_optional_mix_inputs_2(self):
             for t in threads:
                 t.join()
             self.check_deferred_exception()
-            self.check_status(self.model_name_, {1: 4}, 4, 4)
+            self.check_status(self.check_status_model, {1: 4}, 4, 4)
         except Exception as ex:
             self.assertTrue(False, "unexpected error {}".format(ex))
 
+    def test_ensemble_all_inputs(self):
+        # The ensemble is only a wrapper over 'identity_2_float32'
+        self.model_name_ = 'ensemble_identity_2_float32'
+        self.test_all_inputs()
+        # From the ensemble's perspective, the requests are processed as it is
+        self.check_status(self.model_name_, {1: 2}, 2, 2)
+
+    def test_ensemble_optional_same_input(self):
+        # The ensemble is only a wrapper over 'identity_2_float32'
+        self.model_name_ = 'ensemble_identity_2_float32'
+        self.test_optional_same_input()
+        # From the ensemble's perspective, the requests are processed as it is
+        self.check_status(self.model_name_, {1: 2}, 2, 2)
+
+    def test_ensemble_optional_mix_inputs(self):
+        # The ensemble is only a wrapper over 'identity_2_float32'
+        self.model_name_ = 'ensemble_identity_2_float32'
+        self.test_optional_mix_inputs()
+        # From the ensemble's perspective, the requests are processed as it is
+        self.check_status(self.model_name_, {1: 4}, 4, 4)
+
+    def test_ensemble_optional_mix_inputs_2(self):
+        # The ensemble is only a wrapper over 'identity_2_float32'
+        self.model_name_ = 'ensemble_identity_2_float32'
+        self.test_optional_mix_inputs_2()
+        # From the ensemble's perspective, the requests are processed as it is
+        self.check_status(self.model_name_, {1: 4}, 4, 4)
+
```

Comment:
Do you think it will be useful to have tests on more composite models? Like following ensemble:

 <some_model> -> <identity_2_float32>
 
 the some model can be identity_0_float32, identity_1_float32 ?